src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure/Commands/ASPNetBenchmarks/AspNetBenchmarksAnalyzeCommand.cs

```diff
@@ -38,7 +41,17 @@ public override int Execute([NotNull] CommandContext context, [NotNull] AspNetBe
                 configurationToCommand[line[0]] = line[1];
             }
 
-            Dictionary<string, List<MetricResult>> results = ExecuteAnalysis(configuration, configurationToCommand, new());
+            Dictionary<string, List<MetricResult>> result = ExecuteAnalysis(configuration, configurationToCommand, new());
+            if (result.Count == 0)
+            {
+                AnsiConsole.MarkupLine($"[bold green] No report generated since there were no results to compare. [/]");
+            }
+
+            else
+            {
+                AnsiConsole.MarkupLine($"[bold green] Report generated at: {Path.Combine(configuration.Output.Path, "Results.md")} [/]");
```

Comment:
This report name seems like something that could be in a shared location (and allow some kind of `--report-name <val>` argument if desired)

Reply:
Will consider this for all the test suites in a separate PR.